### PR TITLE
Copy external libraries into external/ 

### PR DESCRIPTION
**Pull request by saneyuki**
## Motivation

It's hard to recognize that it's an external libraries a file which is in managed directories by git.

## Detailed Design

* Copy external libraries into `external/`
   * We may have any other good naming for this library. But I think this is most straight and reasonable at now.

### CODE REVIEW

File: webextensions/background/background.js [hunk 1]
```diff
@@ -250,7 +251,7 @@ export async function tryInitGroupTab(aTab) {
     return;
   browser.tabs.executeScript(aTab.apiTab.id, Object.assign({}, scriptOptions, {
     //file:  '/common/l10n.js'
-    file:  '/resources/l10n.js' // ES module does not supported as a content script...
+    file:  '/external/l10n-classic.js' // ES module does not supported as a content script...
```
Comment (saneyuki) [inline webextensions/background/background.js]: I use _classic_ as the term of traditional script type by [_classic script_](https://html.spec.whatwg.org/multipage/webappapis.html#classic-script) in whatwg html spec

**Comment by piroor** [comment]
This change looks good!

There is one concern: how should we name the directory? For example, Ruby on Rails uses "vendor" for the purpose. I have no knowledge about such a naming on JS products, so I don't know "external" is major or not. The current name "external" looks enough reasonable so I agree to use the name if there is no other known major naming for this purpose. Any advice?

**Comment by saneyuki** [comment]
@piroor 

I chose "external" by these reason:

* I seem _vendor_ means that we commit an external library to VCS.
   * If I commit a file or place sub-modules directly to `/webextensions/`, I may use _vendor_.
* _third_party_ is more mozilla way, but I hesitate to use it for this project which is purely JS project.
* With a JavaScipt programmer hat, in today's JavaScript project,  I use npm to place an external libraries and it's rare case to vendor them without npm or other package system.
* Submodules are placed in _extlib_. So I seem _external_ or _lib_ might be suitable. I also think that using _extlib_ in `/webextensions/` would be confusable.

I don't have any strong opinion for that. If there is no other reasonable opinion, you can choose a like one.

**Comment by piroor** [comment]
Thanks. Hmm, then I think I should use `extlib` for the purpose (because it clearly indicates that there are "libraries" from "external repository") and move current submodules to another place like `submodules`. I'll do that after merging this.

**Pull request merged**